### PR TITLE
Reset student numbers

### PR DESCRIPTION
**Pull request by timhodson**
@ichilvers Creating a pull request for easy sharing of comments etc.
This pull request would pull from the forked repo back into Joe's repo once the PR is 'approved' for merging into the master branch of the main tool.

### CODE REVIEW

**Comment by ichilvers** [comment]
Highlighting this now following conversation about readiness of this branch for review on 30th June in Community Dev meet.

Ran this as test against our rolled over lists and whilst there were no issues with the code, there is one scenario that the script doesn't currently take into account:

Where a hierarchy node never had student numbers in the past and doesn't have student numbers currently, but the list does have student numbers (in the override_student_numbers), which have been added to the list manually.

Apparent from the log (created by functions format_old_node_data() and format_new_node_data()):
e.g.
Old Node Relationships
mt1cc (22); mt2cc (57); mtmcc (3)

New Node Relationships
mt1cc (null); mt2cc (37); mtmcc (null)

In this scenario, there are questions about what should be done. Our preference (UoR) is to keep the existing student numbers as we find overall they are a better indicator of actual numbers for the current year than 0.

I will update the script to take this scenario into account.

File: reset_student_numbers/src/reset_numbers.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+						<option value="2">WARNING</option>
+						<option value="1">ERROR</option>
+						<option value="3">INFO</option>
+						<option value="4">DEBUG</option>
+					</select>
+					<br/><br/>
+					<input type="checkbox" name="DRY_RUN" value="writeToLive">
+					<label for="dryrun">Tick this box to write to live tenancy</label><br>
+					<input type="checkbox" name="OVERRIDE" value="overrideDefault">
+					<label for="override">Tick this box to override the default behaviour (see above)</label><br>
```
Comment (timhodson) [inline reset_student_numbers/src/reset_numbers.html:63]:
```suggestion
					<label for="override">Don't keep list student numbers if hierarchy student numbers are zero (see above)</label><br>
```

File: reset_student_numbers/src/reset_numbers.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+				Select logging level: 
+					<select name="loglvl">
+						<option value="2">WARNING</option>
+						<option value="1">ERROR</option>
+						<option value="3">INFO</option>
+						<option value="4">DEBUG</option>
+					</select>
+					<br/><br/>
+					<input type="checkbox" name="DRY_RUN" value="writeToLive">
+					<label for="dryrun">Tick this box to write to live tenancy</label><br>
```
Comment (timhodson) [inline reset_student_numbers/src/reset_numbers.html:61]:
```suggestion
					<label for="dryrun">Write to live tenancy?</label><br>
```

**Review by timhodson — COMMENTED** [review]
Looks good to me. A few minor comments. 

Because I authored this pull request I can't approve it, so we'll need @jmunzer to approve changes before merging.

**Pull request merged**